### ISSUE TITLE
Feature Request: The Voten Bookmarklet

### ISSUE THREAD

**Issue by mustafabozkurt** [voten-co/voten#34]
Hi Voten!
I was thinking about the Feature option for Voten that submitting a story by clicking the bookmarklet in your browser. So You can easily submit your stories to Voten with just few clicks.

I think it would be useful for Voten users. What do you think?

**Comment by sullyfischer** [comment]
Hi Mustafa

You mean like a chrome extension?

**Comment by mustafabozkurt** [comment]
Hi @sullyfischer. Unlike a traditional browser plug-in, a bookmarklet is not downloaded or installed, nor does it require time to load each time the browser is launched. To the end user, bookmarklets look like regular bookmarks that go across their browser toolbar. When selected, the bookmarklet opens a pop-up window instead of taking the user to a website to access the application.

It would be a simple plugin that displays a bookmarklet link for users to drag to their browser toolbar or add to their favorites. They can click that link when browsing other webpages to open a new window at post submission process. In other words, a bookmarklet is a tool for making it easier to use your site.

**Comment by toneywu** [comment]
In WEIBO, which is a Chinese version of twitter. the is a similar Bookmarklet function.

You don't need to install any extension,  just the the save the Bookmarklet on the browser tool bar.
when you are reading a website, just click the Bookmarket icon on the browser-toolbar. a new page will be popped to abstract content from the page you are viewing.

I noticed the following code is listed in the Sina WEIBO Bookmarklet toolbar icon, (instead of a URL)

javascript:(function(){window.open('http://v.t.sina.com.cn/share/share.php?title='+encodeURIComponent(document.title)+'&url='+encodeURIComponent(location.href)+'&source=bookmark','_blank','width=450,height=400');})()

for example
1. I am viewing a Voten.co
2. By clicking the toolbar icon a new window popped up.
3. after some processing in the newly-popped window, I get the following URL in the newly-popped window

http://service.weibo.com/share/share.php?title=Voten%20-%20Social%20Bookmarking%20For%20The%2021st%20Century&url=https%3A%2F%2Fvoten.co%2F&source=bookmark

**Comment by yoginth** [comment]
Yeah, it will be useful (like https://getpocket.com/chrome/ uses!)

**Comment by sullyfischer** [comment]
@yoginth that's a great example. I will write one for sure.

**Comment by yoginth** [comment]
Thanks

**Comment by YoginthS** [comment]
Eagerly waiting for this! it's much important!
